grpc-interface-support/test/integration/test_integration_cpp.py

```diff
@@ -86,7 +92,7 @@ def test__integration():
     print("============= BUILDING CLIENT ===================")
     os.chdir(os.environ["SERVICE_CLIENT_ROOT"])
     ensure_project_initialized()
-    ensure_package_is_generated()
+    ensure_package_is_generated(2)
```

Comment:
What is the meaning of the magic number "2" here ( and "1" a few lines above). I.e. how do we know that the results will exist at index 2, or is that just based on trial and error ("os.scandir in our system gives wanted directory at position 2")

Reply:
Its a temporary workaround since we know the paths at which the CI generates/places the projects. CLI uses a hash of the absolute project path to generate the project cache path. We can remove the index with a better strategy once the CLI provides a command to return the path.

Reply:
Changed the code to use the new CLI command